### PR TITLE
Add Azure Workload Identity upgrade ci.

### PR DESCRIPTION
**Pull request by huangmingxia**
[CCO-405](https://issues.redhat.com/browse/CCO-405)

### CODE REVIEW

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-4.14-upgrade-from-devpreview-4.14-azure-ipi-workload-identity-tp-f28

**Comment by huangmingxia** [comment]
@jianping-shu @jianlinliu @yunjiang29 @jhou1 could you help review the upgrade ci when you have time? much thaanks.

**Comment by huangmingxia** [comment]
@jhou1 could you help review and approve this upgrade ci ^^

File: ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/tp/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+workflow:
+  as:  cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/tp/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-workflow.yaml:2]: If I remember right, this file was already merged, why this PR create it again?

Comment (huangmingxia) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/tp/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-workflow.yaml:2]: yes, updating

Comment (huangmingxia) [reply ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/cco-manual-workload-identity/tp/cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-tp-workflow.yaml:2]: It seems that I used a different repo locally to submit the code to cause this problem.

**Comment by huangmingxia** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-4.14-upgrade-from-devpreview-4.14-azure-ipi-workload-identity-tp-f28

**Comment by huangmingxia** [comment]
Hi @jianlinliu @yunjiang29 @jhou1 could you help review and approval this pr? Much thanks.

**Comment by huangmingxia** [comment]
Hi @liangxia could you help approval this pr? thanks.

**Pull request merged**